src/main/java/macrobase/analysis/summary/Summary.java

```diff
@@ -0,0 +1,7 @@
+package macrobase.analysis.summary;
+
+/**
+ * Represents a summarization result, currently an itemset with a score and weight.
+ */
+public abstract class Summary {
```

Comment:
I'm open to suggestions here too. Summary should have a defined interface, but I'm not sure what exactly is needed from a Summary. `List<Datum> getMembers()`? `getWeight()`?